pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerGauge.java

```diff
@@ -42,6 +42,11 @@ public enum BrokerGauge implements AbstractMetrics.Gauge {
   NETTY_POOLED_ARENAS_DIRECT("arenas", true),
   NETTY_POOLED_ARENAS_HEAP("arenas", true),
 
+  /**
+   * The queue size of ServerRoutingStatsManager main executor service.
+   */
+  STATS_MANAGER_QUEUE_SIZE("statsManagerQueueSize", true),
```

Comment:
Rename to `ROUTING_STATS_MANAGER_QUEUE_SIZE` for clarity

Reply:
addressed